souto-app/src/components/ticketDescription.js

```diff
@@ -157,17 +160,20 @@ export default class TicketDescription extends Component {
                 value={this.state.description}
                 fullWidth
                 onChange={(event) => {
-                  this.setState({ description: event.target.value })
+                  this.setState({ description: event.target.value });
                 }}
                 multiline
                 rows={6}
                 rowsMax={6}
+                inputProps={{
+                  maxLength: 80,
```

Comment:
Here the user suppose to write a description. 80 chars is very samll. Chane it to 300